pkg/nuclide-outline-view/lib/rpc-types.js

```diff
@@ -13,6 +13,7 @@ import type {TokenizedText} from '../../commons-node/tokenizedText-rpc-types';
 
 export type OutlineTree = {
   icon?: string, // from atom$Octicon (that type's not allowed over rpc so we use string)
+  kind?: string, // kind you can pass to the UI for theming, from OutlineTreeKind
```

Comment:
I'm not sure what restrictions your RPC stuff has. The one above is supposed to be from the list of atom$Octicons but is also defined as string as "that type's not allowed over rpc" - do you have any docs about the RPC process or steps I can test this with?